democracy/views/section.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = SectionImage
-        fields = ['title', 'url', 'width', 'height', 'caption', 'image']
+        fields = ['title', 'url', 'width', 'height', 'caption', 'image', 'ordering']
```

Comment:
Image order wasn't saved to db because ordering field was missing from serializer, so seems like all section images in production have default order of one (1). This should be checked and fixed if so.

Reply:
I believe the numbers in production are alright, because multiple images haven't really been POSTed yet through the API.

However, I think this field should only be present at the root level image endpoint. That is because in the section json, the image array should always be ordered according to this field. Similarly, the field should be set when POSTing inlined images according to image order in the POSTed array.

Granted, it's stupid to have the field only in some views, but in most cases it is indeed redundant, as we are only dealing with lists of images within one section.